### ISSUE TITLE
FireFox speed adjustment nolonger works.

### ISSUE THREAD

**Issue by BloodAsp** [codebicycle/videospeed#124]
I just checked, it says there was an update 6h ago, which is consistent with noticing that it worked yesterday, but no longer today.
If there is anything I can do to help, let me know as I am unsure.

**Comment by codebicycle** [comment]
Please give more context, like website, Firefox version, OS, etc.

**Comment by BloodAsp** [comment]
Hmm, well I'm gonna blame YouTube as it started working. ¯\_(ツ)_/¯ 

It was on Youtube, Firefox: 85.0 (64-bit), Ubuntu 20.04.2 LTS x86_64.  I had full system resources available pretty much.
The symptom is that a hotkey I have setup to go 2.5 speed wouldn't speed the video up, or change the ui to display a change.
I tried to rule out the hotkey, so at the time I tried it with the +/- buttons on the gui which also did not work.

I checked now, and all of the above is now working, sorry for the ephemeral ticket.  Thank you for the literally life time saving tool!
